### PR TITLE
Add openshift persister and test also batch saver strategy

### PR DESCRIPTION
**Pull request by Ladas**
Depends on:
- [x] https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/73
- [x] https://github.com/ManageIQ/manageiq-providers-openshift/pull/34
Add openshift persister and test also batch saver strategy

### CODE REVIEW

File: app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb [hunk 1]
```diff
@@ -15,12 +15,8 @@ def ems_inv_to_hashes(inventory, options = Config::Options.new)
         @data
       end
 
-      def initialize_inventory_collections(ems, options)
-        super
-        # get_container_images=false mode is a stopgap to speed up refresh by reducing functionality.  When it's flipped from true to false, we should at least retain existing metadata.
-        if options.get_container_images
-          initialize_custom_attributes_collections(ems.container_images, %w(labels docker_labels))
```
Comment (cben) [inline app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb]: this is still needed, unless I missed it somewhere else

Comment (Ladas) [inline app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb]: it is inside of the OpenShift persistor

**Comment by cben** [comment]
LGTM :+1:
#34 has landed, you can rebase.

**Comment by Ladas** [comment]
@cben it should be ready now

**Pull request merged**